fairscale/optim/__init__.py

```diff
@@ -19,3 +19,7 @@
     from .grad_scaler import GradScaler
 except ImportError:
     logging.warning("Torch AMP is not available on this platform")
+try:
+    from .madgrad import MADGRAD
+except ImportError:  # pragma: no cover
```

Comment:
is this catch needed? for adascale we just directly import it without the catch.